DTS-STN/Alpha-Site
# Default Layout

# Description

[Default Layout](https://trello.com/c/yLIvs2oL/89-default-layout)

This PR includes a new `Layout` component to be used on pages as well as a new molecule `EntryBanner` which is the `Banner` component combined with the `ActionButton` configurations previously found on the homepage.

The `Layout` component contains the Canada.ca top-level header and footer seen here:

Header:

![image](https://user-images.githubusercontent.com/31868510/113020836-fcd28b80-9150-11eb-8aad-2247f57fe2b2.png)

Footer:

![image](https://user-images.githubusercontent.com/31868510/113020977-1ffd3b00-9151-11eb-85fb-759cf4214203.png)

Additionally, the `Layout` component has an optional prop called _banner_ that will take an element and render it above the main children of the `Layout` component. This way banners or other full bleed elements can be passed into a specific page's layout and use it's own margins.

## Acceptance Criteria

There should be a default layout for all pages on the site that reflects Canada.ca design guidelines.

## Test Instructions

1. `npm run storybook`
2. view Layout component in storybook to see responsive margins on content

## Help Requested

- Anything you think I could do to improve this

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
